src/prasad/Bankingsystem.java

```diff
@@ -0,0 +1,70 @@
+package prasad;
+
+public class Bankingsystem {
+	
+	int accountbalance ;
+	int debitamount;
+	int creditamount;
+	int debitcount;
+	int creditcount;
+	int printbalance;
+	static int totaldebitcount;
+	static int totalcreditcount;
+	static int totalprintbalancecount;
+	
+	void creditamt(int amt) {
+		creditamount= creditamount+ amt;
+		accountbalance = accountbalance+amt;
+		creditcount++;
+		totalcreditcount++;
+	}
+	
+	void debitamt(int amt) {
+		debitamount=debitamount+amt;
```

Comment:
why this operation ?

Reply:
I have added this operation to calculate amount being debited from the operation so that I can display that amount in SOP. Similar operation is included to record credited amount.